From b7817dfcac99fc23993be54fe1a6fbbe3416bc79 Mon Sep 17 00:00:00 2001
From: Joel Stanley <joel@jms.id.au>
Date: Tue, 20 Oct 2015 15:01:06 +1030
Subject: [PATCH 7/7] Release 4.4.3-openpower2

 - ast: Switch SCU to VGA output on POST
 - scsi: ignore errors from scsi_dh_add_device()
 - mlx4: Set UAR page size to 4KB regardless of system page size

Signed-off-by: Joel Stanley <joel@jms.id.au>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 802be10c40c5..898d37d2de72 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 4
 PATCHLEVEL = 4
 SUBLEVEL = 3
-EXTRAVERSION =
+EXTRAVERSION = -openpower2
 NAME = Blurry Fish Butt
 
 # *DOCUMENTATION*
-- 
2.7.0

